package.json

```diff
@@ -8,10 +8,10 @@
     "packages/*"
   ],
   "scripts": {
-    "build": "turbo run build",
+    "build": "NODE_ENV=production && turbo run build",
     "changeset": "changeset",
     "commit": "cz",
-    "dev": "turbo run dev --parallel",
+    "dev": "NODE_ENV=development && turbo run build:lib && turbo run dev --parallel",
```

Comment:
this should solve the race condition we notice when running the `turbo dev` (not locating the tokens)